app/elements/feedback-page/feedback-page.html

```diff
@@ -239,28 +239,7 @@
                 properties: {
                     user: {
                         type: Object,
-                        value: function() {
-                            return {
-                                firstName: FIRST_NAME,
-                                lastName: LAST_NAME,
-                                country: COUNTRY,
-                                companyName: COMPANY_NAME,
-                                numberOfPeople: NUMBER_OF_PEOPLE,
-                                numberOfServers: NUMBER_OF_SERVERS,
-                                discover: null,
-                                hosted: [],
-                                hasExtraHosted: false,
-                                extraHosted: null,
-                                hasExtraDiscover: false,
-                                extraDiscover: null,
-                                satisfaction: null,
-                                alternative: null,
-                                hasExtraAlternative: false,
-                                extraAlternative: null,
-                                improve: null
-                            };
-                        },
-                        notify: true
+                        computed: "_computeUser(user)"
```

Comment:
Unfortunately you can't since some extra properties are added in there which are not present in socket data, so we need to declare them for dual data binding.